### ISSUE TITLE
Unusable on Emacs 27.0.50: "Unknown rx symbol"

### ISSUE THREAD

**Issue by alienbogart** [angelic-sedition/emacs-sentence-navigation#9]
### My Environment

- **OS**: MX Linux 18
- **$ uname -r**: 4.19.0-5-amd64
- [dotfiles](https://github.com/mrbig033/dotfiles)
- **i3wm** version 4.13
- [~/.config/i3](https://github.com/mrbig033/dotfiles/blob/master/i3/config)
- **GNU Emacs** 27.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.11) of 2019-11-19
- [~/.emacs.d](https://github.com/mrbig033/emacs02)

### Issue

All `sentence-nav` commands are broken and not executed at all. I only this message: `rx--translate-symbol: Unknown rx symbol ‘maybe-sentence-end’`.

**Comment by noctuid** [comment]
Yeah, I have the same issue. I'm guessing [ample-regexps](https://github.com/immerrr/ample-regexps.el) is broken with Emacs 27.

**Comment by tillheilmann** [comment]
Any chance of this issue being addressed in the near future? Emacs' forward-sentence is driving me mad and I would love to give this a try. (If I had more than just very VERY basic Lisp knowledge I would try to help, sorry.)

**Comment by noctuid** [comment]
It looks like initial emacs 27 support was added for ample-regexps a couple days ago. You can try updating ample-regexps and maybe rebuilding sentence-navigation and see if it works. Even then, this package needs work. I'm planning rewrite it completely on top of [things](https://github.com/noctuid/things.el), but things is nowhere near finished, and I won't have enough free time to finish it anytime soon. If you find any minor bugs in sentence-navigation, I may fix them if I have time.
